docs/community/related_projects.rst

```diff
@@ -0,0 +1,78 @@
+.. _relatedprojects :
+
+Related Projects
+================
+
+On this page we list a number of projects that are
+affiliated with or makes use of QCoDeS in some way or another.
+
+If you would like to get your project featured here please submit a
+pull request adding your project to this page.
+The page is located at ``docs\community\related_projects.rst`` within the QCoDeS
+repository.
+
+
+QCoDeS contrib drivers
+----------------------
+
+QCoDeS contrib drivers is a collection of drivers for many
+different instruments developed by the community. If you have
+developed a driver that your wish to share with the community
+this is the ideal place for it.
+
+Broadbean can be installed with pip from PyPi
+with ``pip install qcodes_contrib_drivers``
+The source code is hosted on Github
+`here <https://github.com/qcodes/qcodes_contrib_drivers>`__
+and the documentation can be found
+`here <https://qcodes.github.io/Qcodes_contrib_drivers/>`__
+
+
+Broadbean
+---------
+
+Broadbean is a library for making pulses. Intended to be used
+with QCoDeS (and in particular its Tektronix AWG 5014 driver)
+
+Broadbean can be installed with pip from PyPi
+with ``pip install broadbean``
+The source code is hosted on Github
+`here <https://github.com/qcodes/broadbean>`__
+and the documentation can be found
+`here <https://qcodes.github.io/broadbean/>`__
+
+
+Plottr
+------
+
+Plottr is a framework for inspecting data, based on flowcharts from pyqtgraph.
+plottr's main aim is to allow the user to define custom data processing
+flows and plotting. A particular use case is data filtering and plotting.
+Plottr has dedicated support for plotting QCoDeS datasets.
+
+
+Plottr can be installed with pip from PyPi
+with ``pip install plottr``
+The source code is hosted on Github
+`here <https://github.com/toolsforexperiments/plottr/>`__
+and the documentation can be found
+`here <https://plottr.readthedocs.io/en/latest/>`__
+
+zhinst-qcodes
+-------------
+
+The Zurich Instruments drivers for QCoDeS (zhinst-qcodes) is a package of
+instrument drivers for devices produced by Zurich Instruments for QCoDeS.
+The QCoDeS instrument drivers are based on the Zurich Instruments Toolkit.
+
+zhinst-qcodes can be installed with pip from PyPi
+with ``pip install zhinst-qcodes``
+The source code is hosted on Github
+`here <https://docs.zhinst.com/zhinst-qcodes/en/latest/>`__
+and the documentation can be found
+`here <https://github.com/zhinst/zhinst-qcodes/>`__
+
+
+
+
```

Comment:
do we want to mention https://github.com/QuTech-Delft/qtt and https://github.com/QuTech-Delft/qilib ? or we'll let them mention themselves if they so wish?

Reply:
just added a comment above to add just qtt